main/lib/src/Event.cc

```diff
@@ -14,7 +14,11 @@ namespace eudaq {
       "EORE",
       "HITS",
       "FAKE",
-      "SIMU"
+      "SIMU",
+      "EUDAQ2",
+      "PACKET",
```

Comment:
this flag is the replacement for the packet class in eudaq 2.0. therefore it has to stay in